README.md

```diff
@@ -1,2 +1,30 @@
 # agent-packaging
-Combined packaging scripts for the Stackdriver monitoring agent
+
+Combined packaging scripts for the Stackdriver monitoring agent.
+
+## Branches in Use
+
+* `master`
+
+  We use this branch to build the `6.X.X` agent DEB and RPM packages for the Cloud Rapture repos.
+
+* `stackdriver-agent-5.5.2`
+
+  We use this branch to build the `5.2.2-X` agent DEB and RPM packages for the Cloud Rapture repos.
+
+* `legacy-agent-rpm`
+
+  We use this branch to build the RPM package for the legacy repo.stackdriver.com repo.
+
+* `legacy-agent-deb`
+
+  We use this branch to build the DEB package for the legacy repo.stackdriver.com repo.
+
+* `test-containers`
+
+  We use this branch to store `Dockerfile`s that are used to build base containers from different distros
+  with necessary dependencies installed so agents can be built in that environment.
```

Comment:
Oops, just realized that this is actually describing the `build` directory. The `test-containers` branch will contain containers for reproducible testing of third-party application support.

Reply:
Done